Mathlib/Data/Real/EReal.lean

```diff
@@ -445,6 +445,62 @@ theorem eq_bot_iff_forall_lt (x : EReal) : x = ⊥ ↔ ∀ y : ℝ, x < (y : ERe
     exact ⟨x.toReal, coe_toReal_le h⟩
 #align ereal.eq_bot_iff_forall_lt EReal.eq_bot_iff_forall_lt
 
+/-! ### Intervals and coercion from reals -/
+
+lemma exists_between_ofReal_left {x : EReal} {z : ℝ} (h : x < z) : ∃ y : ℝ, x < y ∧ y < z := by
```

Comment:
Why not this instead of these 2 lemmas?
```
lemma exists_between_ofReal {x z : EReal} (h : x < z) : ∃ y : ℝ, x < y ∧ y < z := by
```

Reply:
Because I want one of the inequalities to be between reals and not reals coerced to `EReal`s.

Reply:
But it certainly makes sense to add your version, too.

Reply:
Your two lemmas are both an `EReal.coe_lt_coe_iff` rewrite away from Yury's lemma. That seems better to me.